src/components/Portfolio/Portfolio.styles.ts

```diff
@@ -90,37 +92,27 @@ export const SmallAvatar = styled(Avatar)(({ theme }) => ({
   border: `2px solid ${theme.palette.surface2.main}`,
 }));
 
-const circleAnimation = keyframes`
-  0% {
-    stroke-dashoffset: 129;
-    transform: rotate(0);
-  }
-  50% {
-    stroke-dashoffset: 56;
-    transform: rotate(45deg);
-  };
-  100% {
-    stroke-dashoffset: 129;
-    transform: rotate(360deg);
-  }
-`;
+export const Icon = styled(SvgIcon)``;
 
-export const CircularProgressPending = styled(MuiCircularProgress)`
-  color: ${({ theme }) =>
-    theme.palette.mode === 'light'
-      ? theme.palette.primary.main
-      : theme.palette.primary.light};
-  animation-duration: 3s;
-  position: absolute;
-  .${circularProgressClasses.circle} {
-    animation-duration: 2s;
-    animation-timing-function: linear;
-    animation-name: ${circleAnimation};
-    stroke-dasharray: 129;
-    stroke-dashoffset: 129;
-    stroke-linecap: round;
-    transform-origin: 100% 100%;
-  }
-`;
+export const CustomDivider = styled(Divider)(({ theme }) => ({
+  backgroundColor: theme.palette.surface2.main,
+  opacity: 0.3,
+}));
 
-export const Icon = styled(SvgIcon)``;
+export const PortfolioBox = styled(Box)(({ theme }) => ({
+  flexDirection: 'column',
+  display: 'flex',
+  justifyContent: 'center',
+  paddingX: '8px',
+}));
+
+export const NoTokenImageBox = styled(Box)(({ theme }) => ({
+  backgroundColor: 'grey',
+  borderRadisu: '50%',
+  height: 40,
+  width: 40,
+  display: 'flex',
+  justifyContent: 'center',
+  alignContent: 'center',
+  alignItems: 'center',
+}));
```

Comment:
Not 100% knowing, but could this also be replaced by a Skeleton?

Reply:
it is not a skeleton but a Box when no image.